tests/test_ote_training.py

```diff
@@ -0,0 +1,359 @@
+        test_data_root=osp.join(dataset_params.images_test_dir),
+        test_ann_file=osp.join(dataset_params.annotations_test))
+    labels_schema = generate_label_schema(dataset.get_labels(), dataset.is_multilabel())
+    return dataset, labels_schema
+
+def get_default_test_action_classes() -> List[Type[BaseOTETestAction]]:
+    return [
+        ClassificationTestTrainingAction,
+        OTETestTrainingEvaluationAction,
+        OTETestExportAction,
+        OTETestExportEvaluationAction,
+        OTETestPotAction,
+        OTETestPotEvaluationAction,
+        OTETestNNCFAction,
+        OTETestNNCFEvaluationAction,
+        OTETestNNCFExportAction,
+        OTETestNNCFExportEvaluationAction,
+    ]
+
+class ClassificationTrainingTestParameters(DefaultOTETestCreationParametersInterface):
+    def test_case_class(self) -> Type[OTETestCaseInterface]:
+        return generate_ote_integration_test_case_class(
+            get_default_test_action_classes()
+        )
+
+    def test_bunches(self) -> List[Dict[str, Any]]:
+        test_bunches = [
+                dict(
+                    model_name=[
+                       'Custom_Image_Classification_EfficinetNet-B0',
+                    ],
+                    dataset_name='lg_chem_short',
+                    usecase='precommit',
+                ),
+                dict(
+                    model_name=[
+                       'Custom_Image_Classification_EfficinetNet-B0',
+                    ],
+                    dataset_name=['lg_chem','cifar100'],
+                    max_num_epochs=KEEP_CONFIG_FIELD_VALUE,
+                    batch_size=KEEP_CONFIG_FIELD_VALUE,
+                    usecase=REALLIFE_USECASE_CONSTANT,
+                ),
+        ]
+        return deepcopy(test_bunches)
+
+    def short_test_parameters_names_for_generating_id(self) -> OrderedDict:
+        DEFAULT_SHORT_TEST_PARAMETERS_NAMES_FOR_GENERATING_ID = OrderedDict(
+            [
+                ("test_stage", "ACTION"),
+                ("model_name", "model"),
+                ("dataset_name", "dataset"),
+                ("max_num_epochs", "num_epochs"),
+                ("batch_size", "batch"),
+                ("usecase", "usecase"),
+            ]
+        )
+        return deepcopy(DEFAULT_SHORT_TEST_PARAMETERS_NAMES_FOR_GENERATING_ID)
+
+    def test_parameters_defining_test_case_behavior(self) -> List[str]:
+        DEFAULT_TEST_PARAMETERS_DEFINING_IMPL_BEHAVIOR = [
+            "model_name",
+            "dataset_name",
+            "max_num_epochs",
+            "batch_size",
+        ]
+        return deepcopy(DEFAULT_TEST_PARAMETERS_DEFINING_IMPL_BEHAVIOR)
+    
+    def default_test_parameters(self) -> Dict[str, Any]:
+        DEFAULT_TEST_PARAMETERS = {
+            "max_num_epochs": 1,
+            "batch_size": 2,
+        }
+        return deepcopy(DEFAULT_TEST_PARAMETERS)
+
+
+class ClassificationTestTrainingAction(OTETestTrainingAction):
```

Comment:
```suggestion

# TODO: This function copies with minor change OTETestTrainingAction
#             from ote_sdk.test_suite.
#             Try to avoid copying of code.
class ClassificationTestTrainingAction(OTETestTrainingAction):
```